src/main/java/org/wso2/carbon/module/swiftiso20022/models/mt103models/blocks/Block01.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.models.mt103models.blocks;
+
+/**
+ * Class that models request payload block01.
+ */
+public class Block01 {
+    static final String BLOCK_NAME = "block01";
```

Comment:
Isn't this already available in the MT103Constants class? Check other places too

Reply:
Add class modifiers to all models in commit 4e687a247135e6bc9d43274b4f9f97756cf64480